### PR TITLE
added page for hashicorp-vault

### PR DESCRIPTION
**Pull request by jesse-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/vault-integration/docs/

File: _docs/integrations/hashicorp-vault.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Hashicorp Vault"
+description: "How to use secrets from Vault in your Codefresh pipelines"
+group: integrations
+toc: true
+---
+
+Codefresh can use secrets from your Hashicorp Vault installation. This way you have full control over secret storage and rotation.
+
+`Note: This feature is for Enterprise accounts only.`
```
Comment (kostis-codefresh) [inline _docs/integrations/hashicorp-vault.md:10]: Use > syntax for this. See https://daringfireball.net/projects/markdown/syntax#blockquote

File: _docs/integrations/hashicorp-vault.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Hashicorp Vault"
+description: "How to use secrets from Vault in your Codefresh pipelines"
```
Comment (kostis-codefresh) [inline _docs/integrations/hashicorp-vault.md:3]: This is a new feature. We need an entry in the what's new page.

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/vault-integration/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/vault-integration/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/vault-integration/docs/

**Pull request merged**